eng/pipelines/runtimelab.yml

```diff
@@ -119,108 +101,3 @@ stages:
-        jobTemplate: /eng/pipelines/common/templates/runtimes/build-test-job.yml
-        buildConfig: Checked
-        platforms:
-        - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
-        jobParameters:
-          testGroup: innerloop
-          liveLibrariesBuildConfig: Release
-          dependsOn:
-          - build_Linux_x64_Checked_
-          - build_Linux_x64_Release_
-
-    #
-    # CoreCLR Test executions using live libraries
-    #
-    - template: /eng/pipelines/common/platform-matrix.yml
-      parameters:
-        jobTemplate: /eng/pipelines/common/templates/runtimes/run-test-job.yml
-        buildConfig: Checked
-        platforms:
-        - Linux_x64
-        helixQueueGroup: pr
-        helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
-        jobParameters:
-          testGroup: innerloop
-          liveLibrariesBuildConfig: Release
-          dependsOn:
-          - coreclr_common_test_build_p0_AnyOS_AnyCPU_Checked
-
-    - template: /eng/pipelines/common/platform-matrix.yml
-      parameters:
-        jobTemplate: /eng/pipelines/common/templates/runtimes/run-test-job.yml
-        buildConfig: Checked
-        platforms:
-        - Windows_NT_x64
-        helixQueueGroup: pr
-        helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
-        jobParameters:
-          testGroup: innerloop
-          liveLibrariesBuildConfig: Release
-          dependsOn:
-          - coreclr_common_test_build_p0_AnyOS_AnyCPU_Checked
-          - build_Windows_NT_x64_Checked_
-          - build_Windows_NT_x64_Release_
-
-    #
-    # Libraries Release Test Execution against a release coreclr runtime
-    #
-    - template: /eng/pipelines/common/platform-matrix.yml
-      parameters:
-        jobTemplate: /eng/pipelines/libraries/run-test-job.yml
-        buildConfig: Release
-        platforms:
-        - Linux_x64
-        helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
-        jobParameters:
-          isFullMatrix: false
-          isOfficialBuild: false
-          testScope: innerloop
-          liveRuntimeBuildConfig: Release
-          dependsOnTestBuildConfiguration: Release
-          dependsOnTestArchitecture: x64
-          dependsOn:
-          - build_Linux_x64_Release_
-
-    - template: /eng/pipelines/common/platform-matrix.yml
-      parameters:
-        jobTemplate: /eng/pipelines/libraries/run-test-job.yml
-        buildConfig: Release
-        platforms:
-        - Windows_NT_x64
-        helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
-        jobParameters:
-          isFullMatrix: false
-          isOfficialBuild: false
-          testScope: innerloop
-          liveRuntimeBuildConfig: Release
-          dependsOnTestBuildConfiguration: Release
-          dependsOnTestArchitecture: x64
-          dependsOn:
-          - build_Windows_NT_x64_Release_
-
-- ${{ if eq(variables.isOfficialBuild, true) }}:
```

Comment:
You need this for official builds.

Reply:
And also, it might be worth trying an official build before merging if you're interested on them as well.

Reply:
how do I make official build?

Reply:
You need to push a branch into this repo: https://dnceng.visualstudio.com/internal/_git/dotnet-runtimelab

Then queue a build, using that branch as the source branch on this pipeline: https://dnceng.visualstudio.com/internal/_build?definitionId=895